frontend/src/views/Transactions/Transactions.jsx

```diff
@@ -105,6 +106,13 @@ export const Transactions = () => {
     }
   }
 
+  const apiEndpoint = useMemo(() => {
```

Comment:
Hi @hamed-valiollahi , just a suggestion we could use the filtering model of ag-grid to filter out the organizations like below. You can refer this in ViewOrganization component.
`  useEffect(() => {
    if (gridRef.current) {
      // clear any previous filters
      localStorage.removeItem(`${gridKey}-filter`)
      const statusFilter = gridRef?.current?.api?.getFilterInstance('isActive')
      if (statusFilter) {
        statusFilter.setModel({
          type: 'equals',
          filter: showActive ? 'Active' : 'Inactive'
        })
      }
      gridRef?.current?.api?.onFilterChanged()
    }
  }, [showActive])`

Reply:
@prv-proton, Hi Prashanth, Thanks for reviewing my PR and for your valuable feedback. The approach you suggested is definitely better and follows best practices. Initially, I wanted to use AGGrid filtering to avoid changes to the backend. However, I found out that AGGrid can't use an OR condition between two columns. I need to filter transactions where either `toOrganizationId` or `fromOrganizationId` matches the selected organization. Since you have more experience with AGGrid, please correct me if I'm wrong. Thank you

Reply:
Yes, you're right in the backend I've implemented and_ conditions only. But you can create your own custom fitlers as well. Probably in future you can refer to 
/lcfs/backend/lcfs/web/api/user/repo.py (apply_filters method)

Reply:
@prv-proton, Thanks Prashanth for your comment. I mean that when I want to pass the `toOrganizationId` and `fromOrganizationId` from JavaScript to follow AGGrid filtering, I need to use [Advanced Filtering](https://www.ag-grid.com/javascript-data-grid/filter-advanced/), which is part of a [paid plan](https://www.ag-grid.com/license-pricing/). Also, another issue I've faced is that we don't have any `toOrganizationId` or `fromOrganizationId` columns available for filtering.

Example:
```js
const initialAdvancedFilterModel: AdvancedFilterModel = {
  filterType: "join",
  type: "AND",
  conditions: [
    {
      filterType: "join",
      type: "OR",
      conditions: [
        {
          filterType: "number",
          colId: "age",
          type: "greaterThan",
          filter: 23,
        },
        {
          filterType: "text",
          colId: "sport",
          type: "endsWith",
          filter: "ing",
        },
      ],
    },
    {
      filterType: "text",
      colId: "country",
      type: "contains",
      filter: "united",
    },
  ],
};
```

Reply:
@hamed-valiollahi , this is just a filter schema which you'll pass from the frontend, and backend will decide how to handle the filtering as we're doing server-side rendering. If the columns aren't present then you can include them and hide by setting a flag in column definition.

Reply:
@prv-proton , Awesome! Thanks a lot, Prashanth, for the clarification. I'll apply the changes. I appreciate you taking the time to leave comments and review my PR. 🙏